tests/muxcable_test.py

```diff
@@ -416,6 +421,72 @@ def test_config_muxcable_tabular_port_with_incorrect_port(self):
 
         assert(result.exit_code == 1)
 
+    @mock.patch('os.geteuid', mock.MagicMock(return_value=0))
+    @mock.patch('sonic_y_cable.y_cable.get_eye_info', mock.MagicMock(return_value=[0, 0]))
+    def test_show_muxcable_eye_info(self):
+        runner = CliRunner()
+        db = Db()
+
+        result = runner.invoke(show.cli.commands["muxcable"].commands["eyeinfo"],
+                               ["0", "0"], obj=db)
+
+        assert(result.exit_code == 0)
```

Comment:
In Python, `assert` is a keyword, not a function. Suggest adding a space after `assert`, and the parentheses are not technically needed, either. I realize that I missed mentioning this in an earlier PR, so it's OK to leave these for now and open a separate PR later to change all of them. It's up to you.

Reply:
fixed